instance/models/instance.py

```diff
@@ -87,6 +88,7 @@ class Instance(ValidateModelMixin, TimeStampedModel):
     ACTIVE = 'active'
     BOOTED = 'booted'
     PROVISIONED = 'provisioned'
+    PROVISIONING_FAILURE = 'provisioning_failure'
```

Comment:
Use of a noun instead of an adjective/adverb

Also, would this be the best way to define an error state? To add error handling for the other states, we would then need to duplicate each individual state, to add a `..._FAILED` state for each of them. Maybe a separate field would be better? Ie something that could have the values `RUNNING`, `DONE`, `FAILED` could be useful. It would also allow to have a more rubust and maintainable test in https://github.com/open-craft/opencraft/pull/13/files#diff-772e1d190b3c56d74ec801abe130ba8cR50